### ISSUE TITLE
Custom seeking control and listen event listener

### ISSUE THREAD

**Issue by Razinsky** [justinmc/react-audio-player#38]
I have a custom audio player on top of your wrapper. When the user clicks somewhere on the progress bar, I manually update the audio player `currentTime` property so it jumps to the right place in the track. However, looking at the source code, I don't really know why the listen event is cleared? The intended behavior would be to keep broadcasting the onListen event and just continue playing the track.

Am I missing something here or is it a proper/better way you would suggest to work with a custom seeking control?

Thanks by the way for this simple wrapper!

**Comment by Razinsky** [comment]
Thanks @justinmc! I ended up pausing right before changing the current time and playing right after and it works fine. The only thing is the play/pause button now switches so quickly that it looks like a glitch so I need to set a delay before updating the UI which is a bit annoying ;)

**Comment by justinmc** [comment]
That's definitely not an ideal setup, sorry to force a hack on you like that.

I went ahead and removed the clearing of play in `seeked` as mentioned.  Here's the new release: https://github.com/justinmc/react-audio-player/releases/tag/v0.6.2

Actually the `onListen` callback was working fine when the user seeks via the UI by dragging the progress bar.  It was getting cleared by `seeked`, but then immediately afterwards the `play` event set it again.  So the behavior was that the timeout would just restart when the user seeked, which is probably desired.

However, I tried programmatically setting `currentTime` as you mentioned and then `play` is not fired, so it gets permanently cleared.  Removing the suggested line seemed to make it work fine in both cases.

All seems good now.  Though, this got me thinking about if there is a nice React way to handle `currentTime` or not...  I'm not sure but let me know if you have an opinion.
